tests/integration/requirements.txt

```diff
@@ -1,3 +1,6 @@
+boto3
+botocore
+docker
 juju>=2,<3
```

Comment:
juju=3.0.4 instead? Is the last version 3 without breaking changes

Reply:
Fixed